### PR TITLE
Bugfix/fix initial score and final score (#34)

### PR DESCRIPTION
**Pull request by ErisApps**
This PR adds the `cutDistanceScore` to both the `initialScore` and `finalScore` fields.
It also exposes the `cutDistanceScore` itself.

Notes:
- This PR will probably result in merge conflict if PR #36 is merged first or the other way. So let me know if I need to rebase.
- The version still needs to be bumped. But I leave it up to you to decide if I should do it in this PR or not.

### CODE REVIEW

File: BeatSaberHTTPStatus/GameStatus.cs [hunk 1]
```diff
@@ -155,6 +156,7 @@ public class GameStatus {
 			this.wasCutTooSoon = false;
 			this.initialScore = -1;
 			this.finalScore = -1;
+			public int cutDistanceScore = -1;
```
Comment (opl-) [inline BeatSaberHTTPStatus/GameStatus.cs:159]: That's... now how this goes. :P

Comment (JensSchadron) [reply BeatSaberHTTPStatus/GameStatus.cs:159]: Whoops, will fix it in a bit.

Comment (ErisApps) [reply BeatSaberHTTPStatus/GameStatus.cs:159]: My bad, will fix it in a bit.

**Comment by opl-** [comment]
Oops, it fixes #34. My bad. Thanks for the PR.

**Pull request merged**